src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -501,6 +512,15 @@ private void updateButtons() {
             downloadButton.setVisible(true);
             cancelDownloadButton.setVisible(false);
         }
+
+        // if less than 200MB available
+        if (downloadDirectory.getUsableSpace() <= 200 * 1000000) {
```

Comment:
You can use something like 
```Java
private static final long MB = 1024L * 1024;
private static final long MINIMUM_FREE_SPACE_IN_MB = 200 * MB;
```
or just the `MB` if that's the only location where it is used. If it happens to be used more often, we could aim for a `ByteUnit` enum containing these constants...